checkov/kubernetes/graph_builder/graph_components/edge_builders/LabelSelectorEdgeBuilder.py

```diff
@@ -1,16 +1,17 @@
 from __future__ import annotations
 
-from checkov.kubernetes.graph_builder.graph_components.K8SEdgeBuilder import K8SEdgeBuilder
+from checkov.kubernetes.graph_builder.graph_components.edge_builders.K8SEdgeBuilder import K8SEdgeBuilder
 from checkov.kubernetes.graph_builder.graph_components.blocks import KubernetesBlock
+from checkov.kubernetes.kubernetes_utils import FILTERED_RESOURCES_FOR_EDGE_BUILDERS
 
 
 class LabelSelectorEdgeBuilder(K8SEdgeBuilder):
 
     @staticmethod
     def should_search_for_edges(vertex: KubernetesBlock) -> bool:
-        if vertex.metadata is not None and vertex.metadata.labels is not None:
-            return True
-        return False
+        return vertex.metadata is not None \
+            and vertex.metadata.labels is not None \
```

Comment:
will the metadata always include labels?

Reply:
yes, it's initialized with the Metadata class